devicon.json

```diff
@@ -2925,6 +2925,27 @@
         "color": "#F15524",
         "aliases": []
     },
+    {
+        "name": "qt",
+        "tags": [
+            "framework"
+        ],
+        "versions": {
+            "svg": [
+                "original"
```

Comment:
Please add a one-color version of the logo (called the `plain` version). This is required for the font versions.

Reply:
I was confused by this part. The original logo as seen on the OP is already one-color, so should I really duplicate the svg to call it plain or can I get away with the aliases? I was going for the latter.

Reply:
Hmm, I didn't notice that. In that case, we should use aliases instead.